code/modules/projectiles/guns/flamer.dm

```diff
@@ -79,16 +79,16 @@
 	return TRUE
 
 
-/obj/item/weapon/gun/flamer/Fire(atom/target, mob/living/user, params, reflex)
+/obj/item/weapon/gun/flamer/Fire()
 	set waitfor = 0
```

Comment:
I should remove that, no? Since it's a signal handler

Reply:
yes